bundle/kubernetes/0.1.0/wavefront-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,295 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "wavefront.com/v1alpha1",
+          "kind": "WavefrontProxy",
+          "metadata": {
+            "name": "wavefront-proxy",
+            "namespace": "wavefront"
+          },
+          "spec": {
+            "token": "\u003cTOKEN\u003e",
+            "url": "\u003chttps://YOUR_CLUSTER.wavefront.com/api/\u003e",
+            "image": "wavefronthq/proxy:5.7",
+            "enableAutoUpgrade": true
+          }
+        },
+        {
+          "apiVersion": "wavefront.com/v1alpha1",
+          "kind": "WavefrontCollector",
+          "metadata": {
+            "name": "wavefront-collector",
+            "namespace": "wavefront"
+          },
+          "spec": {
+            "daemon": true,
+            "enableAutoUpgrade": true, 
+            "env": [
+              {
+                "name": "HOST_PROC",
+                "value": "/host/proc"
+              },
+              {
+                "name": "POD_NODE_NAME",
+                "valueFrom": {
+                  "fieldRef": {
+                    "apiVersion": "v1",
+                    "fieldPath": "spec.nodeName"
+                  }
+                }
+              },
+              {
+                "name": "POD_NAMESPACE_NAME",
+                "valueFrom": {
+                  "fieldRef": {
+                    "apiVersion": "v1",
+                    "fieldPath": "metadata.namespace"
+                  }
+                }
+              }
+            ],
+            "image": "wavefronthq/wavefront-kubernetes-collector:1.0.3",
+            "tolerations": [
+              {
+                "effect": "NoSchedule",
+                "key": "node.alpha.kubernetes.io/role",
+                "operator": "Exists"
+              },
+              {
+                "effect": "NoSchedule",
+                "key": "node-role.kubernetes.io/master",
+                "operator": "Exists"
+              }
+            ]
+          }
+        }
+      ]
+    capabilities: Seamless Upgrades
```

Comment:
I remember discussing something on the lines of "Seamless Upgrades means we need to support upgrades in openshift environment? " . So just double checking.

Reply:
just saw the remained of the review , Guess this is specifically for kubernetes and we have a separate section/file to define capabilities for openshift.

Reply:
This csv is for kubernetes and we have different file for openshift.  I guess "Seamless upgrade" is supported in k8s deployment.